CONTRIBUTING.md

```diff
@@ -254,3 +254,7 @@ mind.
     [application page](https://keras.io/api/applications/) updating the
     "Available Models" section. The contribution guide for keras.io can be found
     [here](https://github.com/keras-team/keras-io/blob/master/contributor_guide.md).
+-   As every PR requires several CPU/GPU hours of CI testing, we discourage
```

Comment:
Is this strictly about build time or tests execution time?

Reply:
Yea, I think this more for better utilizing the computation resource for building/testing, and also reviewers attention.